### ISSUE TITLE
Book descriptions do not respect paragraph breaks

### ISSUE THREAD

**Issue by kopischke** [bookwyrm-social/bookwyrm#1349]
**Describe the bug**
Paragraph breaks in book descriptions formatted with Markdown are ignored when displayed on page (possible regression as a side effect of #1346?).

**To Reproduce**
1. Go to https://bookwyrm.social/book/27025
2. Note how the description runs as a single paragraph

**Expected behavior**
Paragraphs should be displayed as distinct paragraphs.

**Source**
Retrieved by editing on the displayed page:

```markdown
**_‘My mother used to say I was born reaching, which is true. She also used to say it would get me killed, which it hasn't. Not yet, anyway.’_**

Born in the dirt of the wasteland, Cara has fought her entire life just to survive. Now she has done the impossible, and landed herself a comfortable life on the lower levels of the wealthy and walled-off Wiley City. So long as she can keep her head down and avoid trouble, she's on a sure path to citizenship and security – on this world, at least.

**Of the 380 realities that have been unlocked, Cara is dead in all but 8.**

Cara’s parallel selves are exceptionally good at dying – from disease, turf wars, or vendettas they couldn't outrun – which makes Cara wary, and valuable. Because while multiverse travel is possible, no one can visit a world in which their counterpart is still alive. And no one has fewer counterparts than Cara.

But then one of her eight doppelgängers dies under mysterious circumstances, and Cara is plunged into a new world with an old secret. What she discovers will connect her past and future in ways she never could have imagined – and reveal her own role in a plot that endangers not just her earth, but the entire multiverse.
```

**Instance**
bookwyrm.social

**Additional context**
I tried the “reload without content blockers” trick to bust the cache, but the display remained unchanged.

---

**Tablet:**
    - Device: iPad Pro
    - OS: iOS 14.7.1
    - Browser: Safari
    - Version: N/A

**Comment by mouse-reeve** [comment]
Thank you! This should be fixed shortly by #1346

**Comment by kopischke** [comment]
Ah, I see, I had assumed #1346 was live already. Closing, then.
